### DISCUSSION TITLE
Add new environment variable

### DISCUSSION THREAD

**Discussion by stephanebachelier** [vercel/vercel#6656] [Ideas]
It would be great if any deployed application could have access to the project name and organization. 

I've a project with multiple applications with git and I want to use the "git" url instead of the preview url with a development id as I can't connect app together easily. It's related to the issue with next-auth redirection in a preview environment.

I've found that the "git" url follows this pattern `https://<project>-git-<branch>-<org>.vercel.app`, where :
 - `<project>` is the [project name](https://vercel.com/docs/platform/projects#project-name)
 - `<branch>` is a the url friendly name of the git branch,
 - `<org>` is the name of the Vercel Team or personal account.

I can inject for all the deployed applications the env variables for both `project` and `org` but it would be easier to have it injected by VERCEL, as e.g. VERCEL_PROJECT and VERCEL_TEAM.

**Comment by mathieutu** [comment]
Linked issue:
https://github.com/auth0/nextjs-auth0/pull/440#issuecomment-915923435
https://github.com/auth0/nextjs-auth0/issues/417
